articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -113,6 +113,48 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     config = Configurator(settings=settings)
     ```
 
+## Tracking FastAPI applications
+
```

Comment:
Can you add an excerpt here saying an extension for FastAPI does not exist in OpenCensus, but users can write their own middleware to track calls?

Reply:
Thank you, updated. let me know if further change is required.